src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -480,7 +479,7 @@ Domain<3> BinaryCompactObject::create_domain() const {
         distorted_to_inertial_block_maps{number_of_blocks_};
 
     CubicScaleMap expansion_map{
-        expansion_map_outer_boundary_, expansion_function_of_time_name_,
+        outer_radius_, expansion_function_of_time_name_,
```

Comment:
@geoffrey4444 could you confirm that setting the expansion map outer boundary always to the domain outer boundary is what you want?